### ISSUE TITLE
Saving entries

### ISSUE THREAD

**Issue by park353** [CenterForDigitalHumanities/deer-LR#262]
Entries can be saved with minimal to no information. It would be useful for us to have some minimal information necessary before saving. I'm sure we'll have to think through what that minimal information would be for each entry.

**Comment by thehabes** [comment]
Yes, this will have to be an itemized list of what fields are required in what forms, and what counts as a minimal value for those fields (labels should be greater than 1 character and less than 100 and only contain alpha-numeric characters...and so forth).
